expression/builtin_miscellaneous_vec.go

```diff
@@ -57,3 +57,29 @@ func (b *builtinInetNtoaSig) vecEvalString(input *chunk.Chunk, result *chunk.Col
 func (b *builtinInetNtoaSig) vectorized() bool {
 	return true
 }
+
+func (b *builtinIsIPv4Sig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+	result.ResizeInt64(n, false)
+	i64s := result.Int64s()
+	for i := 0; i < n; i++ {
+		if isIPv4(buf.GetString(i)) {
```

Comment:
Please use `buf.IsNull(i)` to check if this row is `null`.

Reply:
Done.

I still don't feel like we should set `result`'s null bit mask here when `buf.IsNull(i)` is `true`, otherwise the unit test breaks. Please see my previous reply to @zz-jason for more context.

Reply:
After investigating the implementation of `builtinIsIPv4Sig.evalInt` and testing in MySQL, I think you are right.
It's my mistake, please remove this `buf.IsNull` check.~

Reply:
Reverted :)